apps/files/tests/test_utils_.py

```diff
@@ -251,8 +251,8 @@ def test_instanciate_without_data(self):
         with NamedTemporaryFile() as file_:
             file_.write(json.dumps(data))
             file_.flush()
-            pje = PackageJSONExtractor(file_.name)
-            assert pje.data == data
+            mje = ManifestJSONExtractor(file_.name)
+            assert mje.data == data
```

Comment:
The changes in this file aren't related to this bug, I just stumbled on them and saw they needed fixing. I can split them in another PR if needed.